spec/spec_helper.rb

```diff
@@ -25,22 +39,40 @@
 # CLIENT_DEBUG: Show debug messages from the client.
 #   CLIENT_DEBUG=1
 #
-# MONGODB_URI: Connection string to use instead of 127.0.0.1:27017.
-# Specify RS_ENABLED or SHARDED_ENABLED if connecting to a replica set
-# or a sharded cluster, otherwise the test suite will establish a
-# direct connection to the first host in the URI.
-#   MONGODB_URI=127.0.0.1:27001,127.0.0.1:27002
+# MONGODB_URI: Connection string to use. This must be a valid MongoDB URI;
```

Comment:
I have to wonder if it's worth reducing the number of these variables at some point in the future; it seems to me that `MONGODB_URI` and `TOPOLOGY` should be sufficient. It's out of scope for this ticket though, so we can revisit another time.

Reply:
I intend to redo test suite configuration at a future time. There are several other considerations that need to be addressed, for example some tests connect to sockets directly which can't be expressed with either MONGODB_URI or MONGODB_ADDRESSES.